tests/ui/abi/debug.rs

```diff
@@ -7,8 +7,14 @@
 // This pattern is prepared for when we account for alignment in the niche.
 //@ normalize-stderr: "(valid_range): [1-9]\.\.=(429496729[0-9]|1844674407370955161[0-9])" -> "$1: $$NON_NULL"
 // Some attributes are only computed for release builds:
+//@ add-core-stubs
```

Comment:
You also do not need two instances of `//@ add-core-stubs`. This test was already revised by https://github.com/rust-lang/rust/pull/143329

Reply:
```suggestion
```